nodejs/trivialOdbc/main.js

```diff
@@ -0,0 +1,14 @@
+const odbc = require('odbc');
+
+async function test() {
+  try {
+    const connection = await odbc.connect('DSN=LUGDEMO;DRIVER=IBM i Access ODBC Driver');

```

Comment:
DRIVER=IBM i Access ODBC Driver option can be moved into the .odbcini file right? That would make the connect line cleaner.

Reply:
Yes, if you're using a DSN usually all the configuration would be done in the DSN, not on the connection string (unless you specifically wanted to override something, but I don't see why that would want to be done with the driver name).

Reply:
I have created a new .odbcini with
DRIVER=IBM i Access ODBC Driver

and removed that parameter from the connect string leaving it simply 'DSN=LUGDEMO'